torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1327,9 +1327,20 @@ def small_3d_unique(dtype, device):
 
     samples = []
     # Test case for large tensor.
-    largesample = SampleInput(make_tensor((L**3,), device, dtype,
-                              low=None, high=None,
-                              requires_grad=requires_grad))
+    # We need to make sure that when we perform gradcheck, perturbing input
+    # by eps does not change the sorted order
+    valid_sample = False
+    # NOTE: Make sure that if you change eps, the eps used by
+    # gradcheck must be updated to be at least this value
+    eps = 1e-6
+    while not valid_sample:
```

Comment:
This is not usually the best thing as someone changing L would be welcomed with test that run forever :p

Maybe use arange and then divide the result by 100 to get floating point values that are all far enough appart.

Reply:
Ah good catch. I'm going to use randperm so that it comes pre-shuffled